pkg/proc/stack.go

```diff
@@ -100,9 +100,8 @@ type stackIterator struct {
 	stackBarrierPC uint64
 	stkbar         []savedLR
 
-	// regs is the register set for the next frame, callFrameRegs is the
-	// register set for the call frame of the next frame.
-	regs, callFrameRegs op.DwarfRegisters
+	// regs is the register set for the next frame
```

Comment:
nit: Since we set regs after we iterate, wouldn't it be more accurate to say that it's the register set for the current frame?

Reply:
Done